### ISSUE TITLE
Multiselect Dropdown search

### ISSUE THREAD

**Issue by stevemathers** [Semantic-Org/Semantic-UI#4535]
When you have focus on the dropdown then click away from the dropdown to collapse the list, it auto selects an item even thou you did not click that item.

Example of this behavior can be found here: http://semantic-ui.com/modules/dropdown.html#/definition

I would expect a click away to just close the dropdown, not select something.

Seems to only effect the search, since the non-search multiselect works fine.

Thanks
Love SemanticUI!

**Comment by ludesign** [comment]
This is actually controlled by the setting `forceSelection`  which you can disable. It is enabled by default and will force auto selection if the element loses the focus, probably for people who use tab to autocomplete and move forward to the next field.

[Link to the setting](https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/dropdown.js#L3520)

[Link to the actual check performed](https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/dropdown.js#L961)

**Comment by ocervell** [comment]
I'd say the default should be set to False for forceSelection. It feels very unnatural that something is selected when you want to exit the dropdown ...

**Comment by Panman8201** [comment]
I'm seeing this issue even if an item is selected. Let's say you open the dropdown, select an item, then click away. You'll end up getting two items selected, the one you chose and then the next available item.

**Comment by casper1149** [comment]
$('.ui.dropdown').dropdown({
        forceSelection: false
      });

did not work for me, option is getting selected always when I try to exit the control

**Comment by stevemathers** [comment]
wow, thanks for the response guys. so this option did work for me.  My code:

```
  $scope.find('.dropdown').dropdown({
    placeholder: false,
    forceSelection: false
  });
```

$scope could be a page handle, modal handle etc...depending what I'm doing.

I agree in that it should be set to false as the default.

Thanks!

**Comment by genezys** [comment]
You can disable `forceSelection` by default with:

```js
$.fn.dropdown.settings.forceSelection = false;
```

**Comment by jlukic** [comment]
#4634 

I think im going to change this behavior because its driving everyone crazy.

**Comment by seriorso** [comment]
Another problem that ramain in v.2.2.6  on .ui.search.selection.dropdown with forceSelection = false used with allowAdditions = true.
If I write a new item into control input box that doesn't exist into dropdown items list when the form is submitted the form validation itself fail and the new addition disappear..
..please fix it

Try it:
https://jsfiddle.net/seriorso/mdznhbg7/

**Comment by ultraN** [comment]
Life saver, Thanks!

**Comment by jlukic** [comment]
Fixed in `2.2.8`

**Comment by frei-x** [comment]
I also met
